gui/qt/main_window.py

```diff
@@ -1643,6 +1644,22 @@ def read_send_tab(self):
             self.show_error(_('Payment request has expired'))
             return
         label = self.message_e.text()
+    
+        # Cointext payment request activates if user enters "cointext:" followed by country code and number, fills the amount, and pushes "send"
+        cointextPayTo=self.payto_e.lines()[0].lower()
+        isCointext = re.search("^cointext:",cointextPayTo)
+        if isCointext is not None:
+            cointextSats=self.amount_e.get_amount()
+            cointextPhone=''.join(i for i in cointextPayTo[9:len(cointextPayTo)] if i.isdigit())
+            cointextURL="https://pay.cointext.io/p/"+cointextPhone+"/"+str(cointextSats) 
+            self.amount_e.setFrozen(True) 
+            def get_payment_request_thread():
+                pr=paymentrequest.get_payment_request(cointextURL)
+                self.on_pr(pr)
```

Comment:
You can't call on_pr from a thread man. It touches the GUI and you can't touch the GUI from a thread.